corehq/apps/commtrack/models.py

```diff
@@ -780,10 +782,10 @@ class Meta:
     # TODO none of these properties are supported on mobile currently
     # we need to discuss what will be eventually so we know what we need
     # to support here
-    requested_on = DateTimeProperty()
-    approved_on = DateTimeProperty()
-    fulfilled_on = DateTimeProperty()
-    received_on = DateTimeProperty()
+    requested_on = USecDateTimeProperty()
+    approved_on = USecDateTimeProperty()
+    fulfilled_on = USecDateTimeProperty()
+    received_on = USecDateTimeProperty()
```

Comment:
yowsers i totally forgot about requisitions. I wonder if we should kill them. (emailed a few people to discuss)